src/EmojiCompletionProvider.ts

```diff
@@ -73,9 +78,15 @@ export default class EmojiCompletionProvider implements vscode.CompletionItemPro
 
         const kind = vscode.CompletionItemKind.Text;
         return this.emojiProvider.emojis.map((x) => {
-            const item = new vscode.CompletionItem(`::${x.name} ${x.emoji}`, kind);
-            item.filterText = `::${x.name}`;
-            item.documentation = new vscode.MarkdownString(`# ${x.emoji}`);
+            const item = new vscode.CompletionItem({
+                label: `::${x.name} ${x.emoji}`,
+                description: x.tags.join(',  ')
+            }, kind);
+            item.filterText = `::${x.name} ${x.tags.join(' ')} ${x.description} ${x.category}`;
```

Comment:
I don't think `category` should be included in the filter text as it is shared between too many emoji

Also I don't think `description` should be included because it will also match in some unexpected cases